hubble/relay.go

```diff
@@ -409,3 +410,24 @@ func (k *K8sHubble) createRelayClientCertificate(ctx context.Context) error {
 
 	return nil
 }
+
+func (k *K8sHubble) PortForwardCommand(ctx context.Context) error {
+	cmd := "kubectl"
```

Comment:
There is also https://github.com/justinbarrick/go-k8s-portforward

Reply:
I think this is interesting if we want to build it directly into the Hubble client at some point